packages/opencensus-core/src/trace/instrumentation/base-plugin.ts

```diff
@@ -60,10 +78,59 @@ export abstract class BasePlugin implements types.Plugin {
   // while unpatching is not. It should be the other way around
 
   // tslint:disable:no-any
-  applyPatch(moduleExports: any, tracer: modelTypes.Tracer, version: string):
-      any {
-    this.setPluginContext(moduleExports, tracer, version);
+  applyPatch(
+      moduleExports: any, tracer: modelTypes.Tracer, version: string,
+      basedir?: string): any {
+    this.setPluginContext(moduleExports, tracer, version, basedir);
   }
 
   abstract applyUnpatch(): void;
+
+
+  /**
+   * Load internal files according to version range
+   */
+  private loadInternalFiles(): types.ModuleExports {
+    let result: types.ModuleExports = null;
+    this.logger.debug('loadInternalFiles %o', this.internalFileList);
+    if (this.internalFileList) {
+      Object.keys(this.internalFileList).map(versionRange => {
```

Comment:
Prefer `forEach` over `map` if you are not using the return value.

Reply:
Done!